assets/js/api-fetch-shim.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch__non-shim';
+
+if ( global._apiFetch === undefined ) {
```

Comment:
Is using a global here necessary? We should only need to expose this via a global for e2e (which I see we have added as well). Would be great if these two could share the same code/global.

Reply:
I'm curious as well why we need to export this as a global. The `e2e-utilities` script should do that only for the e2e tests, our production assets shouldn't expose it.

_If_ we really need to, then it should be named `_googlesitekitAPIFetch`.